spec/menu_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'menu'
+
+describe Menu do
+  let(:description) {"Cheeseburger"}
+  let(:price) {4.99}
+  let(:burger_to_s) {"#{description}\t#{price}"}
+  let(:burger) {double :dish, :description => description,
+                              :price => price,
+                              :to_s => burger_to_s }
```

Comment:
Style/BlockEndNewline: Expression at 9, 52 should be on its own line.<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)